style.css

```diff
@@ -0,0 +1,100 @@
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
+
+body {
+  font-family: Inter, Haettenschweiler, "Arial Narrow Bold", sans-serif;
+  background: #e5e5e5;
+}
+
+#skeleton {
+  width: 375px;
+  margin: 0 auto;
+  background: #fff;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-left: 24px;
+  margin-right: 28px;
+  padding-top: 27px;
+}
+
+p#welcome {
+  font-weight: 600;
+  font-size: 20px;
+  line-height: 28px;
+  color: #28352f;
+}
+
+#menu {
+  width: 16px;
+  height: 10px;
+}
+
+#page {
+  background-image: url("/images/skeleton.svg");
```

Comment:
- [ ] Kindly make sure your background image displays properly. There is an error in displaying.
- Your design:
<img width="301" alt="Screenshot 2022-03-17 at 10 55 39 AM" src="https://user-images.githubusercontent.com/70269507/158766243-b082c8e8-b02e-4f75-af43-eb726f664d6d.png">

- Figma design:
<img width="294" alt="Screenshot 2022-03-17 at 11 10 23 AM" src="https://user-images.githubusercontent.com/70269507/158766320-be254eea-d9d6-4f86-baad-de39421994a2.png">